e2e-tests/gef/cypress/e2e/application/application-details.spec.js

```diff
@@ -62,12 +62,8 @@ context('Application Details Page', () => {
     it('displays the correct headings', () => {
       applicationDetails.applicationDetailsPage();
       applicationDetails.captionHeading();
-      applicationDetails
-        .mainHeading()
-        .invoke('text')
-        .then((text) => {
-          expect(text.trim()).to.equal('Application Details');
-        });
+
+      cy.assertText(applicationDetails.mainHeading(), 'Application Details');
```

Comment:
can we add a tech debt ticket for content-strings?

Reply:
Created this ticket => https://ukef-dtfs.atlassian.net/browse/DTFS2-7338